client/src/main/java/io/pravega/client/stream/EventPointer.java

```diff
@@ -45,5 +45,13 @@
     static EventPointer fromBytes(ByteBuffer eventPointer) {
         return EventPointerInternal.fromBytes(eventPointer);
     }
-    
+
+    /**
+     * Returns the stream from which event is read from, fully qualified stream name can be obtained from calling {@link Stream#getScopedName()} }
+     * If no event to read then this will return null.
```

Comment:
This does not apply. If no event is read, these is no event pointer in the first place.

Reply:
Yes, That's right. Corrected Java documentation comment.